models/cable/R/write.config.CABLE.R

```diff
@@ -85,10 +85,10 @@ write.config.CABLE <- function(defaults, trait.values, settings, run.id) {
   
   #-----------------------------------------------------------------------
   ### Edit a templated config file for runs
-  if (!is.null(settings$model$config) && file.exists(settings$model$config)) {
-    config.text <- readLines(con = settings$model$config, n = -1)
+  if (!is.null("cable.nml") && file.exists("cable.nml")) {
+    config.text <- readLines(con = "cable.nml", n = -1)
```

Comment:
oops, these first two lines **SHOULD** be settings$model$config not cable.nml-- that's what allows the user to specify a custom cable.nml in their pecan.xml. By contrast, the system.file command is grabbing the default from the PEcAn.CABLE package itself, so that needs to be a specific file name.  Oh, and don't forget to commit that default cable.nml (the one from CABLE-AUX/offline) to models/cable/inst